src/server/service/kv.rs

```diff
@@ -2576,6 +2612,24 @@ impl HealthFeedbackAttacher {
     }
 
     fn attach_if_needed(&mut self, resp: &mut BatchCommandsResponse) {
+        let mut requested_feedback = None;
+
+        for r in resp.responses.iter_mut().map(|r| &mut r.cmd).flatten() {
+            if let BatchCommandsResponse_Response_oneof_cmd::GetHealthFeedback(r) = r {
+                if requested_feedback.is_none() {
+                    requested_feedback = Some(self.gen_health_feedback_pb());
+                }
```

Comment:
When will this logic be used?

Reply:
There is explicit health fetch request in the batch commands I think.

Reply:
To make it a complete API, `GetHealthFeedbackResponse` carries the health feedback information. We don't actually use it for now, but it makes further potential uses possible. When it's used in batch_commands, the feedback information will also be attached to the `BatchCommandsResponse`, so that it can be easily handled by the batch client, and it's transparent to the batch client whether the feedback is periodically sent or explicitly requested.
Here the code is used to check a `BatchCommandsResponse`, and attach the information if explicitly requested (by checking whether there is one or more `GetHealthFeedbackResponse`).